docs/android/setup.md

```diff
@@ -24,10 +24,13 @@ and the following code:
 </receiver>
 ```
 
-Add the following NuGet package:
-
+Click on **Project>Add NuGet Packages** and find and add the following NuGet package:
+```
 - Xamarin.Android.Support.Compat v7 (min version: 26.0.2-rc1)
 
+- Xamarin.Firebase.Messaging (min version: 57.1104.0-beta1)
+```
```

Comment:
My tests are all negative (app doesn't build) if I don't immediately add all 3 NuGet packages.
We should include all 3 of them even in the basic installation.
Since this inclusion automatically puts the app over the single dex method limit, we should mention that, if the build process brakes after adding the packages, the developer should make sure he has enabled multi-dexing in his Android config.